planning/autoware_trajectory_concatenator/src/trajectory_concatenator_node.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright 2025 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "trajectory_concatenator_node.hpp"
+
+#include "trajectory_concatenator_structs.hpp"
+
+#include <autoware/motion_utils/trajectory/trajectory.hpp>
+#include <autoware_utils_uuid/uuid_helper.hpp>
+#include <builtin_interfaces/msg/detail/duration__builder.hpp>
+#include <rclcpp/duration.hpp>
+#include <rclcpp/logger.hpp>
+#include <rclcpp/logging.hpp>
+#include <rclcpp/time.hpp>
+
+#include <autoware_internal_planning_msgs/msg/detail/candidate_trajectory__builder.hpp>
+#include <autoware_internal_planning_msgs/msg/detail/candidate_trajectory__struct.hpp>
+#include <autoware_planning_msgs/msg/detail/trajectory__struct.hpp>
```

Comment:
```suggestion
#include <autoware_internal_planning_msgs/msg/detail/candidate_trajectory__builder.hpp>
#include <autoware_internal_planning_msgs/msg/detail/candidate_trajectory__struct.hpp>
#include <autoware_planning_msgs/msg/detail/trajectory__struct.hpp>
```
`<autoware_internal_planning_msgs/msg/candidate_trajectories.hpp>` and `<autoware_planning_msgs/msg/trajectory.hpp>` is included in `trajectory_concatenator_node.hpp` so you can remove it.